src/cmd/package.go

```diff
@@ -283,3 +346,17 @@ func bindRemoveFlags() {
 	removeFlags.StringVar(&pkgConfig.DeployOpts.Components, "components", v.GetString(V_PKG_DEPLOY_COMPONENTS), lang.CmdPackageRemoveFlagComponents)
 	_ = packageRemoveCmd.MarkFlagRequired("confirm")
 }
+
+func bindPublishFlags() {
+	publishFlags := packagePublishCmd.Flags()
+
+	v.SetDefault(V_PKG_PUBLISH_CONCURRENCY, 3)
```

Comment:
TODO: this line gets run after `bindCreateFlags`, so `default: 3` is missing from the `--help` of create.